darwin/importer/importer.py

```diff
@@ -865,6 +865,22 @@ def import_annotations(  # noqa: C901
         slot_errors, slot_warnings, annotation_format, console
     )
 
+    if annotation_format == "darwin":
+        dataset.client.load_feature_flags()
+
+        # Check if the flag exists. When the flag is deprecated in the future we will always perform this check
+        static_instance_id_feature_flag_exists = any(
+            feature.name == "STATIC_INSTANCE_ID"
+            for feature in dataset.client.features.get(dataset.team, [])
```

Comment:
why do we need this `static_instance_id_feature_flag_exists`? it exists, so seems this is unneeded

Reply:
We only want to run the check that this PR introduces if:
- 1: The `STATIC_INSTANCE_ID` feature flag exists **and,** it's value for the relevant workspace is `True`, or:
- 2: The flag doesn't exist. At this point, the feature will be globally enabled so we will always want to run the check

If we run the check and either of these things isn't true, imported instance IDs can have multiple values throughout an annotation, so the warnings would be false

Reply:
oooh makes sense thanks!